readme.rst

```diff
@@ -3,63 +3,40 @@ Documentation starter pack
-------------------------------------
+Prerequisites
+-------------
 
-You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in an existing code repository.
+Before you begin, ensure you have the following:
 
-**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
+* A GitHub repository where you want to host your documentation. The recommended approach is to host the documentation alongside your code in a `docs` folder. But a standalone documentation repository is also an option.
+* Git and Bash installed on your system.
 
-Standalone documentation repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To create a standalone documentation repository, clone this starter pack
-repository, `update the configuration <#configure-the-documentation>`_, and
-then commit all files to the documentation repository.
-
-You don't need to move any files, and you don't need to do any special
-configuration on Read the Docs.
-
-Here is one way to do this for newly-created fictional docs repository
-``canonical/alpha-docs``:
-
-.. code-block:: none
+Bootstrap your documentation repository
+---------------------------------------
 
-   git clone git@github.com:canonical/sphinx-docs-starter-pack alpha-docs
-   cd alpha-docs
-   rm -rf .git
-   git init
-   git branch -m main
-   UPDATE THE CONFIGURATION AND BUILD THE DOCS
-   git add -A
-   git commit -m "Import sphinx-docs-starter-pack"
-   git remote add upstream git@github.com:canonical/alpha-docs
-   git push -f upstream main
+Bootstrap your repository by following these steps:
 
-Documentation in a code repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Copy [bootstrap.sh](bootstrap.sh) to your repository's root directory.
+* Run the script: ``./bootstrap.sh``
+* Enter the installation directory when prompted. For standalone repositories, enter ".". For documentation alongside code, enter the folder where your documentation are e.g. "docs".
 
-To add documentation to an existing code repository:
+This Bash script does the following:
 
-#. create a directory called ``docs`` at the root of the code repository
-#. populate the above directory with the contents of the starter pack
-   repository (with the exception of the ``.git`` directory)
-#. copy the file(s) located in the ``docs/.github/workflows`` directory into
-   the code repository's ``.github/workflows`` directory
-#. in the above workflow file(s), change the value of the ``working-directory`` field from ``.`` to ``docs``
-#. in file ``docs/.readthedocs.yaml`` set the following:
+* Clones the starter pack GitHub repository
+* Creates the specified installation directory if necessary
+* Udates working directory paths in workflow files, and updates configuration paths in the .readthedocs.yaml file
```

Comment:
Udates

Reply:
Yep. I really should start running spell check locally.